public/app/features/datasources/settings/ButtonRow.tsx

```diff
@@ -17,6 +17,9 @@ const ButtonRow: FC<Props> = ({ isReadOnly, onDelete, onSubmit, onTest }) => {
       <LinkButton variant="secondary" fill="outline" href={`${config.appSubUrl}/datasources`}>
         Back
       </LinkButton>
+      <LinkButton variant="secondary" fill="solid" href={`${config.appSubUrl}/explore`}>
```

Comment:
This is great, and I think it could be even more helpful if the button selected the datasource as well so that the user can immediately start querying their data.

Reply:
I referred to the panel code and we need panel id to be passed to this page right ?

Reply:
Because we're going from datasource settings -> Explore, there is no panel id so you won't be able to use the panel code as is, but you can use it as a reference to see how it builds the url based on the datasource settings. I think we just need to add the datasource name to the url.

Reply:
done